packages/stores/types.ts

```diff
@@ -308,16 +309,17 @@ type TUiStore = {
     setShouldShowCooldownModal: (value: boolean) => void;
 };
 
-type TPortfolioStore = {
-    active_positions: ProposalOpenContract[];
-    error: TCommonStoreError;
-    getPositionById: (id: number) => ProposalOpenContract;
+export type TPortfolioStore = {
+    active_positions: TPortfolioStore['positions'];
```

Comment:
Will remove🙇🏼‍♀️

Reply:
@farzin-deriv It's become a valid TS code since TypeScript 3.7: https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#more-recursive-type-aliases. Using this type and this approach helps us correctly type `open-positions` file, where there are a lot of TS issues currently in master branch. Could you please suggest where it's better to put `TPortfolioStore` so that I can use it in open-positions file in `reports` package 🙂 ? will it be ok if I put it in `shared`? But then I will need to import it from `shared` to this file in `stores` package where it's also used.